lib/ansible/modules/packaging/os/swupd.py

```diff
@@ -0,0 +1,288 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from ansible.module_utils.basic import AnsibleModule
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: swupd
+short_description: Manages bundles with I(swupd)
+description:
+    - "Manages bundles with the I(swupd) update manager, which is used by the
+      Clear Linux Project for Intel Architecture"
+version_added: "2.3"
+author:
+    - "Alberto Murillo (@albertomurillo)"
+options:
+    name:
+        description:
+            - Name of the bundle to install or remove.
+        required: false
+        default: null
+        aliases: ['bundle']
+    state:
+        description:
+            - Indicates the desired bundle state. C(present) ensures the bundle
+              is installed while C(absent) ensures the bundle is not installed.
+        required: false
+        default: present
+        choices: ["present", "absent"]
+    update:
+        description:
+            - Updates the OS to the latest version
+        required: false
+        default: no
+        aliases: ['upgrade']
+        choices: ['yes', 'no']
+    verify:
+        description:
+            - Verify file system
+        required: false
+        default: null
+        choices: ['yes', 'no']
+    manifest:
+        description:
+            - Manifest to verify against to
+        required: false
+        default: null
+        aliases: ['release', 'version']
+    format:
+        description:
+            - The format suffix for version file downloads
+        required: false
+        default: null
+    url:
+        description:
+            - Overrides both I(contenturl) and I(versionurl)
+        required: false
+        default: null
+    contenturl:
+        description:
+            - url for content file download
+        required: false
+        default: null
+    versionurl:
+        description:
+            - url for version string download
+        required: false
+        default: null
+'''
+
+EXAMPLES = '''
+- name Update the OS to the latest version
+  swupd:
+    update: yes
+
+- name: Installs the "foo" bundle
+  swupd:
+    name: foo
+    state: present
+
+- name: Removes the "foo" bundle
+  swupd:
+    name: foo
+    state: absent
+
+- name: Check integrity of filesystem
+  swupd:
+    verify: yes
+
+- name: Downgrade OS to release 12920
+  swupd:
+    verify: yes
+    manifest: 12920
+'''
+
+FILES_NOT_MATCH = "files did not match"
+FILES_REPLACED = "missing files were replaced"
+FILES_FIXED = "files were fixed"
+FILES_DELETED = "files were deleted"
+
+
+def _get_cmd(module, command):
+    p = module.params
+    cmd = "%s %s" % (SWUPD_CMD, command)
+
+    if p["format"]:
+        cmd += " --format=%s" % (p["format"])
```

Comment:
The the formatting argument is single value, it doesn't have to be in braces:

```
cmd += " --format=%s" % p["format"]
```

It improves the readability slightly. The same should apply across the whole code.